### ISSUE TITLE
Models got fail accuracy on BMG but passed on PVC

### ISSUE THREAD

**Issue by mengfei25** [intel/torch-xpu-ops#1305]
### 🐛 Describe the bug

```
python benchmarks/dynamo/timm_models.py --accuracy --float16 -d xpu -n10 --training --only botnet26t_256 --backend=inductor
```

Category | Model | Accuracy
-- | -- | --
timm_models_float16_training | botnet26t_256 | fail_accuracy
torchbench_float32_inference | detectron2_maskrcnn_r_50_fpn | fail_accuracy
timm_models_amp_fp16_training | fbnetv3_b | fail_accuracy
timm_models_bfloat16_training | fbnetv3_b | fail_accuracy
timm_models_amp_bf16_training | gernet_l | fail_accuracy
torchbench_float16_training | hf_Longformer | fail_accuracy
torchbench_bfloat16_training | hf_Longformer | fail_accuracy

### Versions

Environments:
Name| Version
-- | --
Device | BMG
OS | Ubuntu 24.10
Driver | 24.45.31740
Torch | v2.6.0-rc5
Torch-xpu-ops | release/2.6
Triton | 3.2.0
Transformers | 243e186efbf7fb93328dd6b34927a4e8c8f24395
Torchvision | 0.21.0+xpu
Torchaudio | 2.6.0+xpu
Torchbench | 766a5e3a189384659fd35a68c3b17b88c761aaac
Timms | ac3470188b914c5d7a5058a7e28b9eb685a62427
Bundle | 2025.0.4.20241205

**Comment by daisyden** [comment]
@mengfei25  please help to update the latest status and what is accuracy gap for these tests.

**Comment by chuanqi129** [comment]
@mengfei25 check the status in release 2.7 test report

**Comment by Stonepia** [comment]
From the latest 2.7 test report:
internal wiki link: display/mlpcdlval/%28wip%29+BMG-Conda-windows-pt27_rc1

The following models still fail in accuracy:

```
fbnetv3_b
gernet_l
```

Will take a look at that

**Comment by Stonepia** [comment]
We are having an unalignment with CUDA's test:

https://github.com/pytorch/pytorch/blob/d256b2dcb21f5a97f57b918e3129d0e47116e405/benchmarks/dynamo/common.py#L1709

For the CUDA's behavior, some models are fallback to SGD optimizer. 
By apply the same behavior with above, the `fbnetv3_b` could work.

**Comment by chuanqi129** [comment]
@Stonepia will submit a PR to align with Cuda's optimizer for gernet_l

Will further investigate the root cause of fbnetv3_b
